python/pyiceberg/table/metadata.py

```diff
@@ -270,7 +271,7 @@ def set_sort_orders(cls, data: Dict[str, Any]):
         # This is going to be much nicer as soon as sort-order is an actual pydantic object
         # Probably we'll just create a UNSORTED_ORDER constant then
```

Comment:
Do these two lines still apply now that we have the constants and are using `IcebergBaseModel`?

Reply:
I'll remove this right away, thanks for the sharp eye @kbendick